src/messages/ui.en.json

```diff
@@ -197,7 +197,7 @@
   "connectMB.bluetooth.cancelledConnection": "You didn't choose a micro:bit. Do you want to try again?",
 
   "connectMB.bluetooth.connecting": "Connecting…",
-  "connectMB.bluetooth.invalidPattern": "The pattern you drew is invalid",
+  "connectMB.bluetooth.invalidPattern": "The pattern you drew is invalid.",
```

Comment:
I think we were going to change this to "The pattern you have drawn is invalid."